desk/app/treaty.hoon

```diff
@@ -111,6 +111,17 @@
     ::NOTE  this assumes that all treaties in sovereign are also
     ::      present in the treaties map
     (fact-init:io (treaty-update:cg:cc %ini treaties))^~
+    ::
+      [%treaties @ ~]
+    :_  this
+    =/  =ship  (slav %p i.t.path)
+    =/  alliance  (~(get ju allies) ship)
```

Comment:
This is the case for the scry too, but I'm kind of surprised this isn't doing a `(~(put in alliance) ship)`, to make sure the ship we're actually explicitly asking about is also in there, always.